Client/src/app/components/http.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable()
+export class HttpService {
+  constructor(public http: HttpClient) { }
+  //apiKey = 'AIzaSyDiS4N4g6u1t9_EApKFSgRmWZr4vwJbdW4';
+  apiKey = 'AIzaSyDzvVlr87fMi5kn0O4ooLx5VZru3XRXVzA';
+  searchVideosUrl = 'https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&type=video';
+  serverUrl = 'http://localhost:3000';
+  history;
+
+  async getSearchResults(toSearch) {
```

Comment:
What is the type of results? is it a list of titles or video ids? you should change the function name and code to reflact the answer (: